news/2025/2025-01-16-osumania-7k-world-cup-2025-qualifiers-recap.md

```diff
@@ -0,0 +1,168 @@
+
+With the introduction out of the way, let us get into the nitty-gritty of this year's Qualifiers pool! Stay tuned, as later on we'll get to chat with several members from the mappooling team.
+
+If you're new to the series, fret not! [SurfChu85](https://osu.ppy.sh/users/4469895) and [yukina meng](https://osu.ppy.sh/users/9919550) got you covered with the mappool's basics.
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/Ir8Qk92aaI0" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Mappool recap
+
+[**Download the pool here (32 MB)**](https://osu.ppy.sh/beatmaps/packs/P270)
+
+<div align="center" class="osu-md__paragraph">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://player.twitch.tv/?video=2345049575&parent=osu.ppy.sh&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no"></iframe>
+</div>
+
+![Qualifier mappool table](/wiki/shared/news/2025-01-16-osumania-7k-world-cup-2025-qualifiers-recap/mappool.jpg)
+
+Players were given a warm welcome into the tournament with a fairly standard-sized 8 stage Qualifiers pool. But size isn't what matters for all intents and purposes. From changing the LN release picks to be LN coordination, to introducing one of the most intense hybrid maps to date in a Qualifiers pool. And while the core of the pool remains the same, those teams who were able to adapt to the changes were able to gain an edge over their peers!
+
+Speaking of highlights, let's talk about the star of the pool: [aran - Yoake Mae feat. shully](https://osu.ppy.sh/beatmapsets/2303945#mania/4924806) (LN1) by [_Kobii](https://osu.ppy.sh/users/6209713) ::{ flag=MY }::. Whereas in previous years this would've been a long note pick with a focus on releases, this time around the poolers decided to test players' finger coordination as a whole. Other noteworthy maps include [Mono - skip&step](https://osu.ppy.sh/beatmapsets/2115949#mania/4443569) (RC3) by [Blocko](https://osu.ppy.sh/users/4075092) ::{ flag=US }::, a fun and varied rice experience meant to keep players on their toes — or shall we say, their fingers? Finally, we have [seatrus - Hekikuu no Yakusoku](https://osu.ppy.sh/beatmapsets/2303946#mania/4924807) (HB2) by [Critical_Star](https://osu.ppy.sh/users/3793196) ::{ flag=MY }::. Players got a little of tiebreakers to come, as well as the intensity they should expect.
+
+But we've only just begun, and future mappools are yet to be revealed. Who knows what surprises have the poolers been preparing for us? To wrap things up, have a small fun fact! This mappool was *almost* fully comprised of licensed tracks, with seven of its eight maps using songs from the [featured artist listing](https://osu.ppy.sh/beatmaps/artists).
```

Comment:
weird thing to advertise, especially when the ro32 mappool didn't use all fa songs. you either are or aren't using all licensed songs <.< 

```suggestion
But we've only just begun, and future mappools are yet to be revealed. Who knows what surprises have the poolers been preparing for us?
```

Reply:
It felt like a good detail to point up though. Regardless of the round, the fact stays that the pool is literally almost made of licensed tracks. It should not be a given considering this isn't really the norm for most tournaments.

Reply:
featured artist should be capitalised at least

```suggestion
But we've only just begun, and future mappools are yet to be revealed. Who knows what surprises the poolers have been preparing for us? To wrap things up, here's a small fun fact: this mappool was *almost* fully comprised of licensed tracks, with seven of eight maps using songs from the [Featured Artist listing](https://osu.ppy.sh/beatmaps/artists).
```